README.md

```diff
@@ -113,14 +113,14 @@ At SEEK this is done via [sku] as part of the build process.
 
 ### Assertions
 
-To ensure correct usage of its components, Braid performs some precondition and invariant checking at runtime using the [tiny-invariant] library.
+To ensure correct usage of its components, Braid performs some precondition and invariant checking at runtime using the [assert] library.
 
 To prevent these checks from being included in production builds and disrupting the end user experience, it is recommended that `assert` calls are stripped at build time using the [unassert] library.
 
 At SEEK this is done by [sku] as part of the build process via [Babel] with the [babel-plugin-unassert] plugin.
-The minimum `sku` version that supports stripping assertions from the `tiny-invariant` library is [12.6.0].
+The minimum `sku` version that supports stripping assertions from the `assert` library is [12.6.0].
```

Comment:
This line is also no longer relevant. This version of sku added support for `tiny-invariant`, the `assert` library was much much earlier.

```suggestion
```